src/coverlet.core/Reporters/CoberturaReporter.cs

```diff
@@ -133,28 +133,48 @@ public string Report(CoverageResult result)
             return Encoding.UTF8.GetString(stream.ToArray());
         }
 
-        private static IEnumerable<string> GetRootDirs(Modules modules, bool useSourceLink)
+        private static IEnumerable<string> GetBasePaths(Modules modules, bool useSourceLink)
         {
             if (useSourceLink)
```

Comment:
Please add some comment on the idea behind alg for instance 

```
/*
     Workflow

     Path1 c:\dir\file1.cs
	 Path2 c:\dir\file2.cs
	 Path3 e:\dir\file2.cs
     

     1) Search for root dir 
		c:\ ->	c:\dir\file1.cs
				c:\dir\file2.cs
		e:\ ->	e:\dir\file2.cs
		
	 2) Split path on directory separator i.e. for record c:\ ordered ascending by fragment elements
		 Path1 = [c:|dir|file1.cs]
		 Path2 = [c:|dir|file2.cs]
		 
	 3)  Find longest shared path comparing indexes, list is ordered so the longest absolute path will be max Path[0] length
		 
		 Path1[0] = Path2[0] -> add to final fragment list
		 Path1[n] = PathY[n] -> add to final fragment list
		 ...
		 
     4) Concat created fragment list
*/
```

Reply:
Done